tests/auth_tests/test_validators.py

```diff
@@ -57,6 +57,19 @@ def test_validate_password(self):
     def test_password_changed(self):
         self.assertIsNone(password_changed('password'))
 
+    def test_password_changed_with_custom_validator(self):
+        class Validator:
+            def password_changed(self, password, user):
+                self.call_count = 1
+                self.password = password
+                self.user = user
+
+        validator = Validator()
+        self.assertIsNone(password_changed('password', password_validators=(validator,)))
+        self.assertEqual(validator.call_count, 1)
```

Comment:
`call_count` isn't really necessary any more, as accessing `user` or `password` would be an `AttributeError` if the function hadn't been called, but this is a minor issue

Reply:
Right. I removed it.